amazon-kinesis-client-multilang/src/main/java/software/amazon/kinesis/multilang/config/MultiLangDaemonConfiguration.java

```diff
@@ -73,6 +73,8 @@ public class MultiLangDaemonConfiguration {
     private String applicationName;
 
     private String streamName;
+    private String streamArn;
```

Comment:
Yeah we can do that

Reply:
I don't believe so. 

Reason being...
When we go through the properties to be set in the configuration it checks the object type and attempts to have it as that type. Because of that, streamArn will always be seen as type 'String'. An explicit cast from string to Arn does not work.